admin_pages/general_settings/AdminOptionsSettings.php

```diff
@@ -149,7 +157,7 @@ public function generate()
                             ),
                             'default'         => isset($this->registry->CFG->admin->show_reg_footer)
                                 ? filter_var($this->registry->CFG->admin->show_reg_footer, FILTER_VALIDATE_BOOLEAN)
-                                : true,
+                                : DomainFactory::getEventEspressoCoreDomain()->isCaffeinated(),
```

Comment:
I'd prefer we do `apply_filters(...)` here, and add a filter from within caff. That seems like the more normal way to do things, rather than asking "is this a caff install?" (having code that checks if it's a premium version of the plugin sounds bad, and if we were doing it to outright disable some functionality it would be in violation of the WP.org plugin guidelines, right?)

If that's not possible, this is fine then.

Reply:
oh ya! that's a good point :)  I agree.  I'm not sure if we can do the same for the `EE_Config` default because of loading order for the config but I'll try it  there.

Reply:
> having code that checks if it's a premium version of the plugin sounds bad, and if we were doing it to outright disable some functionality it would be in violation of the WP.org plugin guidelines, right?

Ya on the surface it might look bad, but in this case its actually changing the default for an option (so not disabling the ability to toggle on).  In this case it is an option that wp.org coding guidelines would frown on for being "on" by default.

Reply:
Ya it mostly looks bad at first, but is actually ok I think.